src/magic_spell_effect.cpp

```diff
@@ -233,7 +233,7 @@ std::set<tripoint> spell_effect::spell_effect_line( const spell &, const tripoin
     // Clockwise Perpendicular of Delta vector
     const point delta_perp( -delta.y, delta.x );
 
-    const point abs_delta = abs( delta );
+    const point abs_delta = std::abs( delta );
```

Comment:
That is not in `std`:

https://github.com/CleverRaven/Cataclysm-DDA/blob/e397b562bf40ee83a4fffdd6dde5d884f4f3e438/src/point.h#L314-L317